### PR TITLE
change mempool validation function

### PR DESCRIPTION
**Pull request by tudor-malene**
### Why this change is needed

return more errors when submitting tx on a vaidator node

### What changes were made as part of this PR

invoke a different geth function

### PR checks pre-merging

Please indicate below by ticking the checkbox that you have read and performed the required
[PR checks](https://github.com/ten-protocol/ten-internal/blob/main/dev-ops-docs/dev-pr-checks.md)

- [x] PR checks reviewed and performed

### CODE REVIEW

File: integration/constants.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ const (
 	StartPortWalletExtensionUnitTest = 38000
 	StartPortFaucetUnitTest          = 42000
 	StartPortFaucetHTTPUnitTest      = 48000
-	StartPortTenscanUnitTest         = 52000
-	StartPortTenGatewayUnitTest      = 56000
+	StartPortTenscanUnitTest         = 12000
```
Comment (tudor-malene) [inline integration/constants.go:16]: I noticed that tenscan test failing. Grasping on straws here, I thought of just changing the port.

**Review by BedrockSquirrel — COMMENTED** [review]
LGTM generally, just minor questions

File: go/enclave/nodetype/validator.go [hunk 2]
```diff
@@ -109,6 +109,10 @@ func (val *obsValidator) ExecuteStoredBatches() error {
 				return fmt.Errorf("could not store executed batch %s. Cause: %w", batch.Hash(), err)
 			}
 			val.batchRegistry.OnBatchExecuted(batch, receipts)
+			err = val.mempool.Chain.IngestNewBlock(batch)
```
[the remark this answers is not in the document]
Comment (tudor-malene) [reply go/enclave/nodetype/validator.go:112]: good point

**Review by BedrockSquirrel — APPROVED** [review]
lgtm

**Pull request merged**